r35075 | iive | 2012-08-11 12:42:43 -0400 (Sat, 11 Aug 2012) | 4 lines

Quick Build Fix. FFmpeg CodecID is redefined as AVCodecID.
It needs the FFmpeg includes in all places that use it.


Index: libmpdemux/mp_taglists.h
===================================================================
--- libmpdemux/mp_taglists.h	(revision 35074)
+++ libmpdemux/mp_taglists.h	(revision 35075)
@@ -20,6 +20,8 @@
 #define MPLAYER_MP_TAGLISTS_H
 
 #include <stdint.h>
+#include "libavutil/common.h"
+#include "libavformat/avformat.h"
 
 enum CodecID mp_tag2codec_id(uint32_t tag, int audio);
 uint32_t mp_codec_id2tag(enum CodecID codec_id, uint32_t old_tag, int audio);
